pkgs/development/libraries/glibc/common.nix

```diff
@@ -64,6 +65,25 @@ stdenv.mkDerivation ({
          "/bin:/usr/bin", which is inappropriate on NixOS machines. This
          patch extends the search path by "/run/current-system/sw/bin". */
       ./fix_path_attribute_in_getconf.patch
+
+      /* Allow running with RHEL 6 -like kernels.  The patch adds an exception
+        for glibc to accept 2.6.32 and to tag the ELFs as 2.6.32-compatible
+        (otherwise the loader would refuse libc).
+        Note that glibc will fully work only on their heavily patched kernels
+        and we lose early mismatch detection on 2.6.32.
+
+        On major glibc updates we should check that the patched kernel supports
+        all the required features.  ATM it's verified up to glibc-2.26-115.
+        # HOWTO: check glibc sources for changes in kernel requirements
+        git log -p glibc-2.25.. sysdeps/unix/sysv/linux/x86_64/kernel-features.h sysdeps/unix/sysv/linux/kernel-features.h
```

Comment:
Did you do `git checkout release/2.26/master` before `git log`? Is 115 the number of commits in that branch since `glibc-2.26` tag? Shouldn't `git log -p` be called with `glibc-2.26..`?

Reply:
I checked out the version I updated to:
```
$ git describe
glibc-2.26-115-g633e2f7f3d
```

I wanted to re-check since 2.25, as the kernel support was bumped between 2.25 and 2.26 (and on 2.26.. they probably wouldn't do such large changes anyway).